R/derive_merged.R

```diff
@@ -146,6 +146,16 @@
 #'   )
 #'   ```
 #'
+#' @param relationship Expected merge-relationship between the `by_vars`
+#'   variable(s) in `dataset` and the dataset containing the additional `new_vars`.
+#'
+#'   This argument is passed to the `dplyr::left_join()` function. See
+#'   <https://dplyr.tidyverse.org/reference/mutate-joins.html#arguments> for
+#'   more details.
+#'
+#'   Permitted Values for `relationship`: `"one-to-one"`, `"one-to-many"`,
+#'   `"many-to-one"`, `"many-to-many"`, `NULL`.
```

Comment:
```suggestion
#'   *Permitted Values*: `"one-to-one"`, `"many-to-one"`, `NULL`
```
I would remove the values which don't relate to the purpose of the function because it could confuse users. For example if `"one-to-many"` is permitted, users may assume that the function could be used for one-to-many matches. But this is not true. The number of records of the input dataset is not changed.

Reply:
Updated.